### ISSUE TITLE
Granular email subscribe & unsubscribe

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#1313]
- We should create a system where people can subscribe/unsubscribe from certain email lists
- We can have a "mandatory" list of emails where people cannot unsubscribe from them

@ahmadabugosh @WhyldWanderer @NikolaCreatrix

**Comment by oyealmond** [comment]
hey @ahmadabugosh , now that your on this, is this possible?

**Comment by oyealmond** [comment]
Should I move this to Notion @laurenluz?

**Comment by divine-comedian** [comment]
We have this already under the notifications settings menu on the dapp 
https://giveth.io/notifications/settings

![image](https://github.com/Giveth/giveth-planning/assets/67759413/0780cfdb-967e-481c-85eb-da28e22c23b9)

**Comment by laurenluz** [comment]
yeah but like... we don't direct people there at all from the emails. for example, when I click "unsubscirbe" on an email about a donation being made, I get this: 
![image](https://github.com/Giveth/giveth-planning/assets/75490971/418202f7-989a-4088-802a-ce61d770904b)

no option to resubscribe or manage preferences...

also then, the newsletter, from paragraph, has an entirely OTHER unsupscribe/manage preferences page.

**Comment by laurenluz** [comment]
also even though I just "unsubscribed" from clicking the button in my email... when I do the long unintuitive process of finding the notifcation center to reactivate it... I see this:

![image](https://github.com/Giveth/giveth-planning/assets/75490971/237ea5ad-6533-4338-95f2-ec88e9c4fad9)

so I unsubscribed, but the notificatio center doesn't recognize that.... or I did unsubscribe and now have no way to resubscribe.

there is a lot of incongruency w/ our managing emails & the notification center that would ideally be fixed.

**Comment by divine-comedian** [comment]
I spent some time chatting with Ortto support to figure out how we could do this! Great point bringing this up Lauren. 

When a user unsubcribes from giveth mail they are unsubscribed from EVERYTHING. So if they unsubscribe from the newsletter they won't get any emails from Giveth. Including "transactional" notifications like when they receive a donation or their project status changes, etc.. 

We have a way to manually add someone back to our subscribe list using the Ortto API - https://help.ortto.com/a-257-create-or-update-one-or-more-people-merge

![image](https://github.com/Giveth/giveth-planning/assets/67759413/2e1cf96e-01dd-453c-b55d-964ae32c74f3)

However this is a binary switch and can't be granular according to which emails they want. 

There is an option to mark certain emails we send as "Transactional" meaning they override the users subscription settings and are always sent, no matter if they are unsubscribed from emails. 
![image](https://github.com/Giveth/giveth-planning/assets/67759413/cef0697d-a054-40b4-8436-9e5620763300)

### BUT
 - this feature is not included in the current tier of Ortto we are paying for... the next level is quite a bit more expensive 
![image](https://github.com/Giveth/giveth-planning/assets/67759413/c17c04e3-ee12-4507-8033-006118b3914e)

As of now I'm negotiating if we can get a discount on upgrading... 

If not, we'll have to find a hacky solution with the use of tags or something like that...

**Comment by ahmadabugosh** [comment]
@divine-comedian  I think there is a much easier way, we should be able to do this using the "Preference Center" and having multiple audiences. See this article: https://help.ortto.com/a-118-working-with-subscription-preferences

I looked into it and I think we can do it from here in our account: https://ortto.app/ap2-giveth/setup/preference-center 
<img width="612" alt="Screenshot 2024-03-07 at 11 16 19 AM" src="https://github.com/Giveth/giveth-planning/assets/4855928/0ff3b0b7-1291-4d6a-8091-1b0b3d62db48">

We would just need to create multiple "subscription audiences". 

So for example, we could have one for: 
- Giveth general updates
- QF rounds
- Donations to projects you own 
etc. 

And a user can unsubscribe to just the ones they want.

Let's pass this on to @krmet
